File: pyjs/pyjamas-0.1/pyjs/tests/test032.py
def i(x):
    return x

def test_builtins():
    x = []
    y = callable(x)
    z = map(i, x)
    filter(callable, z)
    dir(x)
    if hasattr(x, "foo"):
        foo = getattr(x, "foo")

test_builtins()

File: pyjs/pyjamas-0.1/pyjs/tests/test032.js
function i(x) {
    return x;
}


function test_builtins() {
    var x = new pyjslib_List([]);
    var y = pyjslib_isFunction(x);
    var z = pyjslib_map(i, x);
    pyjslib_filter(callable, z);
    pyjslib_dir(x);
    if (pyjslib_hasattr(x, "foo")) {
    var foo = pyjslib_getattr(x, "foo");
    }
}


    test_builtins();
